binaries/coordinator/src/lib.rs

```diff
@@ -773,6 +773,42 @@ async fn start_inner(
                                 },
                             ));
                         }
+                        ControlRequest::GetNodeInfo => {
+                            use dora_message::coordinator_to_cli::{NodeInfo, NodeMetricsInfo};
+
+                            let mut node_infos = Vec::new();
+                            for dataflow in running_dataflows.values() {
+                                for (node_id, _node) in &dataflow.nodes {
+                                    // Get the specific daemon this node is running on
+                                    if let Some(daemon_id) = dataflow.node_to_daemon.get(node_id) {
+                                        // Get metrics if available
+                                        let metrics = dataflow.node_metrics.get(node_id).map(|m| {
+                                            NodeMetricsInfo {
+                                                pid: m.pid,
+                                                cpu_usage: m.cpu_usage,
+                                                memory_mb: m.memory_bytes as f64 / 1024.0 / 1024.0,
+                                                disk_read_mb_s: m
+                                                    .disk_read_bytes
+                                                    .map(|b| b as f64 / 1024.0 / 1024.0),
+                                                disk_write_mb_s: m
+                                                    .disk_write_bytes
+                                                    .map(|b| b as f64 / 1024.0 / 1024.0),
```

Comment:
If we report in [megabytes](https://simple.wikipedia.org/wiki/Megabyte) we should divide by 1000, not 1024. Dividing by 1024 results in [mebibytes](https://simple.wikipedia.org/wiki/Mebibyte) (symbol `MiB` instead of `MB`).